hal/lib/athena/HALAthena.cpp

```diff
@@ -5,8 +5,6 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "HAL/HAL.h"
```

Comment:
Yes, I think it would be okay to rename HALAthena.cpp to HAL.cpp, as I believe gradle doesn't confuse files of the same name in different directories, and we'll never build a library file with both the Athena and simulation HAL.cpp files...